pkgs/applications/graphics/imgbrd-grabber/default.nix

```diff
@@ -71,6 +71,10 @@ stdenv.mkDerivation rec {
     sed "s|strict\": true|strict\": false|g" -i ./sites/tsconfig.json
```

Comment:
Can this be removed now?
https://github.com/Bionus/imgbrd-grabber/pull/2848

Reply:
I think so.